transforms/ember-object/__testfixtures__/basic.input.js

```diff
@@ -0,0 +1,19 @@
+import EmberObject from "@ember/object";
+
+/**
+ * Comments
+ */
+const MyObj = EmberObject.extend({
+  someProp: true,
+  biz: baz,
+
+  setFoo(bar = "") {
+    this.foo = bar;
+  },
+
+  setMyprop() {
+    this.set();
+  }
+});
+
+const myObj = MyObj.create();
```

Comment:
Do we want this in here?

Reply:
I don't think so, we should avoid changing any calls to `.create()`, as `new` may be changing soon and is not yet considered public API.

Reply:
Yeah, exactly. This is actually not all that unusual, you can see it as a static class factory method:

```js
class Foo {
  static create(args) {
    let instance = new this();

    Object.assign(instance, args);

    return instance;
  }
}
```